apps/dashboard/src/contract-ui/tabs/split/page.tsx

```diff
@@ -44,9 +45,14 @@ export const ContractSplitPage: React.FC<SplitPageProps> = ({
   const configuredChainsRecord = useSupportedChainsRecord();
   const chainId = useDashboardEVMChainId();
   const chain = chainId ? configuredChainsRecord[chainId] : undefined;
-
   const splitQuery = useSplitData(contractQuery.contract);
-  const nativeBalanceQuery = useBalanceForAddress(contractAddress);
+
+  const convertedChain = chainId ? defineDashboardChain(chainId) : undefined;
```

Comment:
pass the chain as a second param (from line 47) so the RPC gets respected if it is explicitly configured